sysl2/sysl/cmd_import.go

```diff
@@ -61,14 +63,29 @@ func (p *importCmd) Execute(args ExecuteArgs) error {
 	case modeXSD:
 		args.Logger.Infof("Using xsd importer\n")
 		imp = importer.LoadXSDText
+	case modeGrammar:
+		args.Logger.Infof("Using grammar importer\n")
+		p.OutputData.AppName, err = findGrammarType(p.filename)
```

Comment:
Why are you not simply using the app-name flag that is passed in on the command line? (and revert the change to make it optional)

Reply:
The app-name is already available as the grammar name, why not use that?